README.org

```diff
@@ -6,6 +6,7 @@ This is the source repository for the 2017 RoboJackets Software training program
 * Slides
 
 - [[https://robojackets.github.io/software-training/slides/TEST#/sec-title-slide][Test Presentation]]
+- [[https://robojackets.github.io/software-training/slides/github#/sec-title-slide][Github Presentation]]
 
 # TODO maybe host these MD files in an external way other than GH?
 * Docs
```

Comment:
Add to docs as well if you want that to be maintained. 

If you don't want markdown export maybe we would like a website export instead [such as this one](http://clubctrl.com/org/prog/ox-twbs.html)?